test/e2e/utils/datainserter.go

```diff
@@ -83,11 +94,17 @@ func (di *DataInserter) SetClientEndpoints(hosts []string) error {
 }
 
 func (di *DataInserter) Insert() error {
-	framework.Infof("Creating keyspace %q with RF %d", di.keyspace, di.replicationFactor)
+	ss := make([]string, 0)
```

Comment:
```suggestion
	ss := make([]string, len(di.replicationFactor))
```

Reply:
do you mean `ss := make([]string, 0, len(di.replicationFactor))` for capacity?

Reply:
yes, in general I meant preallocation to avoid unnecessary copies.